build/pack.cake

```diff
@@ -53,7 +60,8 @@ Task("Test")
 
                 if (!parameters.IsRunningOnMacOS) {
                     settings.TestAdapterPath = new DirectoryPath(".");
-                    settings.Logger = $"nunit;LogFilePath={MakeAbsolute(new FilePath($"{testResultsPath}{projectName}.results.xml"))}";
+                    var resultsPath = MakeAbsolute(new FilePath($"{testResultsPath}/{projectName}.results.xml"));
```

Comment:
`Path.Combine()` won't be better here?

Reply:
usually in cake they use FilePath instead of Path, as it's much flexible. All the extensions like MakeAbsolute use FilePath or DirectoryPath

Reply:
Yes, but I'm thinking about using `Path.Combine()` (or similar variants) so we don't hard code `/` and other OS-dependent path building code, but instead leave that up to the framework and OS.

Reply:
If I change

Reply:
Inspired from the cake team's script `MakeAbsolute(testResultsPath.CombineWithFilePath("vsix.results.xml"));`